Automattic/wp-calypso
# P2: fix huge notice button in "Accept Invite" page

#### Changes proposed in this Pull Request

* This PR fixes the unintended custom styles for the i18n notice button, by limiting the P2 button style customizations to buttons inside the form.

#### Testing instructions



* To replicate the issue
    * Set your browser's language to something other than English.
    * Open a user invite (email or group invite is okay) in incognito.
    * If i18n options are available, you will see the notice but the dismiss button is huge.

* To test the fix, apply this PR or use the Calypso Live link, follow the same steps above and verify that the notice dismiss is normal-sized.
* Everything else in the page should look okay.


**BEFORE**
<img width="800" alt="Screen Shot 2022-02-12 at 12 44 23 AM" src="https://user-images.githubusercontent.com/730823/153633750-ac75a140-5081-4345-adf9-c996d738ce66.png">

**AFTER**
<img width="800" alt="Screen Shot 2022-02-12 at 12 46 57 AM" src="https://user-images.githubusercontent.com/730823/153633761-96bac403-0e52-466e-868e-d5c319e2d2f5.png">
